Documentation/security/policy/language.rst

```diff
@@ -719,11 +721,15 @@ which is defined as follows:
 
         // PortProtocol specifies an L4 port with an optional transport protocol
         type PortProtocol struct {
-                // Port is an L4 port number. For now the string will be strictly
-                // parsed as a single uint16. In the future, this field may support
-                // ranges in the form "1024-2048"
```

Comment:
I guess we excluded this option in order to be more consistent with k8s APIs?

Reply:
Yes, and it was very difficult to implement.